src/external/analysisConverter.ts

```diff
@@ -48,19 +48,24 @@ const logger = Logger('AnalysisConverter');
 const CONVERTER_CONCURRENT_REQUESTS = 5;
 
 const addMetricsToAlignedReadsFile = async (
-  fileDocument: { [k: string]: RdpcFileDocument[] },
+  fileDocument: RdpcFileDocument,
```

Comment:
this seems like a dramatic type change, care to elaborate?

Reply:
It simplifies the function logic, forces type checking on the calling function.

Reply:
its only called in a single place, within this file. the logic is changed there along with this change.